pkgs/applications/editors/vscode/extensions/ms-vsliveshare.vsliveshare/default.nix

```diff
@@ -1,54 +1,23 @@
-# Based on previous attempts:
-#  -  <https://github.com/msteen/nixos-vsliveshare/blob/master/pkgs/vsliveshare/default.nix>
-#  -  <https://github.com/NixOS/nixpkgs/issues/41189>
-{ lib, gccStdenv, vscode-utils
-, autoPatchelfHook, bash, makeWrapper
-, curl, gcc, libsecret, libunwind, libX11, lttng-ust, util-linux
-, desktop-file-utils, xsel
-}:
+{ lib, vscode-utils, xsel }:
 
-let
-  # https://docs.microsoft.com/en-us/visualstudio/liveshare/reference/linux#install-prerequisites-manually
-  libs = [
-    # Credential Storage
-    libsecret
-
-    # NodeJS
-    libX11
-
-    # https://github.com/flathub/com.visualstudio.code.oss/issues/11#issuecomment-392709170
-    libunwind
-    lttng-ust
-    curl
-
-    # General
-    gcc.cc.lib
-    util-linux # libuuid
-  ];
-
-in ((vscode-utils.override { stdenv = gccStdenv; }).buildVscodeMarketplaceExtension {
+vscode-utils.buildVscodeMarketplaceExtension {
   mktplcRef = {
     name = "vsliveshare";
     publisher = "ms-vsliveshare";
-    version = "1.0.5900";
-    hash = "sha256-syVW/aS2ppJjg4OZaenzGM3lczt+sLy7prwsYFTDl9s=";
+    version = "1.0.5918";
+    hash = "sha256-Tk0mKydUF8M7l7NC9wEA7t2rzJWy/mq4/HvIHI2/ldQ=";
   };
-}).overrideAttrs({ buildInputs ? [], ... }: {
-  buildInputs = buildInputs ++ libs;
 
-  # Using a patch file won't work, because the file changes too often, causing the patch to fail on most updates.
-  # Rather than patching the calls to functions, we modify the functions to return what we want,
-  # which is less likely to break in the future.
   postPatch = ''
     substituteInPlace extension.js \
-      --replace "'xsel'" "'${xsel}/bin/xsel'"
+      --replace-fail '"xsel"' '"${xsel}/bin/xsel"'
   '';
 
   meta = {
-    description = "Live Share lets you achieve greater confidence at speed by streamlining collaborative editing, debugging, and more in real-time during development";
+    description = "Real-time collaborative development for VS Code";
     homepage = "https://aka.ms/vsls-docs";
+    changelog = "https://marketplace.visualstudio.com/items/MS-vsliveshare.vsliveshare/changelog";
```

Comment:
Would you mind following alphabetical order for `meta.changelog`?
Move `changelog` to before `description`?

Btw, `meta.broken` is an exception to alphabetical order to avoid git conflicts.

Reply:
Duplicating my response to your previous review here:

> This isn't true; attributes appear in a logical order in derivations. For instance, typically `pname` and then `version` can be found, then `src`, patches, and so on; and `meta` can be found at the end. Likewise, the meta has its own ordering: `changelog` appears after `homepage` and before `license`; this can be seen [in the Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#var-meta-changelog).

If you check that section of the manual, you can also see that `meta.broken` is listed at the end, as you have mentioned. If you want an example of this, see the [changelog attribute of the Git package](https://github.com/NixOS/nixpkgs/blob/88c619a6a356e641753668987df003f81ab6451a/pkgs/applications/version-management/git/default.nix#L383). I see that other VS Code extensions have the changelog attribute in alphabetical order, but IMO these are the ones getting it wrong, not mine.

Reply:
> Duplicating my response to your previous review here:

Thanks for pointing out that.

> I see that other VS Code extensions have the changelog attribute in alphabetical order, but IMO these are the ones getting it wrong, not mine.

Yes, you may be right.